Source/WebCore/css/CSSSelector.h

```diff
@@ -346,25 +317,22 @@ namespace WebCore {
         void setForPage() { m_isForPage = true; }
 
     private:
-        unsigned m_relation              : 4; // enum RelationType.
-        mutable unsigned m_match         : 4; // enum Match.
-        mutable unsigned m_pseudoType    : 8; // PseudoType.
-        unsigned m_isLastInSelectorList  : 1;
-        unsigned m_isFirstInTagHistory   : 1;
-        unsigned m_isLastInTagHistory    : 1;
-        unsigned m_hasRareData           : 1;
-        unsigned m_hasNameWithCase       : 1;
-        unsigned m_isForPage             : 1;
-        unsigned m_tagIsForNamespaceRule : 1;
-        unsigned m_caseInsensitiveAttributeValueMatching : 1;
-#if !ASSERT_WITH_SECURITY_IMPLICATION_DISABLED
-        unsigned m_destructorHasBeenCalled : 1;
-#endif
+        unsigned m_relation : 4 { DescendantSpace }; // enum RelationType.
+        mutable unsigned m_match : 4 { Unknown }; // enum Match.
+        mutable unsigned m_pseudoType : 8 { 0 }; // PseudoType.
+        bool m_isLastInSelectorList : 1 { false };
+        bool m_isFirstInTagHistory : 1 { true };
+        bool m_isLastInTagHistory : 1 { true };
+        bool m_hasRareData : 1 { false };
+        bool m_hasNameWithCase : 1 { false };
+        bool m_isForPage : 1 { false };
+        bool m_tagIsForNamespaceRule : 1 { false };
+        bool m_caseInsensitiveAttributeValueMatching : 1 { false };
+        bool m_destructorHasBeenCalled : 1 { false };
```

Comment:
Any reason why this has to be compiled all the time now?

Reply:
I changed this back.